File: src/config/favorites.ts
/**
 * Default "Pinned" entities — the cards surfaced at the top of the Home screen out of the box.
 *
 * This is the *seed*: the personalization editor (`store/prefsStore.ts` + the Customize screen)
 * lets the user pin/unpin and reorder, persisting their list to localStorage and overriding this
 * default. Order here is the display order; entities not present in the store are skipped.
 *
 * **The alarm panel is deliberately not here**, though it used to be. The Dashboard grew a
 * security hero (`SecurityStatusBar`) whose three big arm discs ARE the alarm control, so pinning
 * the panel put a second Off/Home/Away directly beneath the first. The locks stay, because the
 * hero only *summarizes* them ("Front Door open") — it has no lock or unlock control, so a pinned
 * lock card is the one place on Home you can actually act on one.
 */
export const favorites: string[] = [
  "lock.front_door_lock",
  "lock.back_door_lock",
];

File: android/app/src/main/java/com/hawksnest/config/Favorites.kt
package com.hawksnest.config

/**
 * Default "Home" favorites — the entities surfaced at the top of the Home screen out of the box.
 * Order here is the display order; entities not present in the store are skipped. Ported from
 * `src/config/favorites.ts`.
 *
 * **Since 2026-08-07 this is a seed, not the whole story** (parity with web's
 * `src/store/prefsStore.ts`): the Devices tab's pinned rail reads the user's stored pin list
 * from `util/DevicePrefsStore`, falling back to this list only while that store was never
 * customized (`core/logic/Pins.kt effectivePins` — the first pin/unpin/move materializes the
 * seed). Long-press any device row → Pin / Unpin / Move up / Move down.
 */
val favorites: List<String> = listOf(
    "lock.front_door_lock",
    "lock.back_door_lock",
    "alarm_control_panel.home",
)
